### PR TITLE
Adding support to the LogicApps Workflow Run Delete operation.

### PR DESCRIPTION
**Pull request by aalevato**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5947 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5947/commits/8ecee267d7e4dabbd4de82cb76509c96ee9945c7

File: specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json
Comment (refortie) [inline specification/logic/resource-manager/Microsoft.Logic/preview/2018-07-01-preview/logic.json]: Looks like a copy paste error, I'd change the example name

**Comment by aalevato** [comment]
The "model validation" errors being pointed out are not related to any of my changes here. Please disregard them.

**Review by shahabhijeet — APPROVED** [review]

**Comment by shahabhijeet** [comment]
@sergey-shandar shoudl we be ignoring this error "Bash exited with code '1'." for model validation?

**Comment by aalevato** [comment]
@shahabhijeet @sergey-shandar Can we please complete and merge this PR? Thanks!

**Comment by shahabhijeet** [comment]
@sergey-shandar is this ok to merge this with the failures I see?

**Comment by shahabhijeet** [comment]
@aalevato atthis point I will wait for @sergey-shandar to approve if it's ok to merge this PR

**Comment by aalevato** [comment]
Could someone please approve to merge it already? The Run Delete operation has been up from quite a while, but many customers are not using because the swagger isn’t reflecting it. It’s just a small change. Please help with completing this PR asap, guys.

**Comment by shahabhijeet** [comment]
@aalevato please make sure the required CI jobs are green and there are no merge conflicts

**Comment by yungezz** [comment]
hi @aalevato , are you still on this PR? could you pls resolve conflicts and make sure CI passed, then we can merge it.

**Comment by yungezz** [comment]
@aalevato  ping again

**Comment by aalevato** [comment]
I was oncall last week and couldn't look into it. As soon as I'm done with my current priorities I'll review it (again). However, I must point out that the "Details" links didn't work weeks ago and don't work now (they show a message saying "An Error has Occurred" without details). Plus, please look into the files being merged - I'm just adding documentation and describing the new method we added months ago. I will try again in some days to see if I can do any kind of true investigation on it.

**Comment by yungezz** [comment]
@aalevato can we close the PR for now since it's not ready for merge, and the PR is over 4 months, there're lots of conflict with latest code base already. Pls create new PR when you're ready for review and merge.

**Comment by aalevato** [comment]
@yungezz No, we can't. I will get back to it and check how complex will be to solve the merge conflicts. Hopefully this time someone will be able to read and address my comments in time and help us get the code merged.

**Comment by yungezz** [comment]
@aalevato thanks for taking care of it.  ping us when you're ready for review and merge.

**Comment by aalevato** [comment]
@yungezz  @shahabhijeet  If I click on "Checks" or try to drilldown the "Details" of any check error, I get a 500. Is it happening because of the migration, did anything get corrupted? How can I check the conflicts and edit the files online without it?

**Comment by aalevato** [comment]
@yungezz @shahabhijeet Hey guys - I added a new PR and will abandon this one once merged. Could you please help review and approve the new one? => https://github.com/Azure/azure-rest-api-specs/pull/6994

**Pull request closed without merging**
